lib/KTable/index.vue

```diff
@@ -207,6 +227,12 @@
         return style;
       };
 
+      onMounted(() => {
+        nextTick(() => {
+          debounce(checkScrollable, 100);
```

Comment:
The z-index issues have been resolved! I still see the `isTableScrollable` value as false on scrollable tables. After some investigation I realized that this debounced function needs to be assigned to a variable and then called here for it to eventually run `checkScrollable()`.

```
      const debouncedCheck = debounce(checkScrollable, 100);

      onMounted(() => {
        nextTick(() => {
          debouncedCheck();
        });
      });

```